### ISSUE TITLE
Undetected?

### ISSUE THREAD

**Issue by Nasayuna** [acuifex/acuion#31]
Is this fuzion copy undetected?

or how can I vac bypass in Linux?

**Comment by acuifex** [comment]
i don't think that any of the fuzion forks or even linux cheats are detected, and vac on linux is a joke

**Comment by Wristing** [comment]
i sub to acuifex as well.

**Comment by AdverseMiller** [comment]
There’s literally no reason to worry, as long as you do everything correctly, you will never get detected by vac
